pkg/minishift/registration/redhat_test.go

```diff
@@ -112,22 +104,14 @@ func TestRedHatRegistratorUnregister(t *testing.T) {
 	}
 	commander := provision.GenericSSHCommander{Driver: d}
 	registrator := NewRedHatRegistrator(commander)
-	s.CommandToOutput["sudo subscription-manager version"] = `server type: This system is currently not registered.`
-	if err := registrator.Unregister(param); err != nil {
-		t.Fatal("Distribution shouldn't able to unregister")
-	} else {
-		cmd := "sudo subscription-manager unregister"
-		if _, ok := s.Commands[cmd]; ok{
-			t.Fatalf("Expected command: %s", cmd)
-		}
-	}
+
 	s.CommandToOutput["sudo subscription-manager version"] = `server type: RedHat Subscription Management`
 	if err := registrator.Unregister(param); err != nil {
 		t.Fatal("Distribution should able to unregister")
 	} else {
 		cmd := "sudo subscription-manager unregister"
 		if _, ok := s.Commands[cmd]; !ok{
-			t.Fatalf("Expected command: %s", cmd)
+			t.Fatalf("Expected command: sudo subscription-manager unregister")
```

Comment:
@hferentschik no, so I want to check here what command suppose to get executed so if we change `cmd` variable it will let us show what actual command it should have.